User Engagement Lead/survey_questions.md

```diff
@@ -0,0 +1,79 @@
+# Survey questions
+
+## Readme
+
+This survey aims at evaluating users’ experiences with ESMValTool and
```

Comment:
Can we also articulate some outcomes we want to achieve, something more specific than is in the text?